src/ansys/dpf/post/fluid_simulation.py

```diff
@@ -263,9 +260,8 @@ def _get_result(
             Overrides any other qualifier argument such as `phases`, `species` or `zone_ids`.
         named_selections:
             Named selection or list of named selections to get results for.
-        result_type:
-            A ``result_type_enum`` value to inform a test on requested zones is necessary.
-            An integrated result will for example only be available on faces.
+        integrated:
+            An integrated result cannot be requested on another location than its native_location.
```

Comment:
Shouldn't we mention the actual definition of integrated result? Integrated results in Dpf are indeed able to be averaged. It's Fluids+integrated results the ones that aren't (to be compliant with FBU's directives)

Reply:
@rafacanton could you give me a description of why FBU's directive is to not average integrated fluid results?